codeforamerica/vita-min
# Tbe 53 flakey tests

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/TBE-53
## What was done?
- Everything we found and fixed re: flaky tests is described in the [Flaky Tests investigation doc](https://www.notion.so/cfa/Flaky-Tests-4f3c50c65fac4a2d8115cd7d894f0dbc)
- We also resolved a few warnings being thrown by tests
## How to test?
- CircleCI has been how we've detected flakiness, so the first check should be whether all tests consistently pass on CircleCI. 
- We should also continue to monitor CircleCI to see whether any more/remaining flakiness is reported, and prioritize fixing that as followup
- Risk Assessment
  - We now disable our rate limiter (`Rack::Attack`) in tests, but this shouldn't affect any live traffic environment
